cmd/contour/serve.go

```diff
@@ -151,6 +151,12 @@ type serveContext struct {
 	httpsAddr      string
 	httpsPort      int
 	httpsAccessLog string
+
+	tlsConfig `json:"tls"`
```

Comment:
We should also support flags so that it can be set during deployment.
similar flags: envoy-service-https-port